client/hotrod-client/src/test/java/org/infinispan/client/hotrod/impl/iteration/BaseMultiServerRemoteIteratorTest.java

```diff
@@ -70,7 +80,7 @@ public void testBatchSizes() {
       int maximumBatchSize = 120;
       RemoteCache<Integer, AccountHS> cache = clients.get(0).getCache();
 
-      populateCache(CACHE_SIZE, this::newAccount, cache);
+      populateCache(CACHE_SIZE, org.infinispan.client.hotrod.impl.iteration.Util::newAccount, cache);
```

Comment:
nitpick
```suggestion
      populateCache(CACHE_SIZE, Util::newAccount, cache);
```

same for below

Reply:
We can't do this as we're already importing `org.infinispan.commons.util.Util`.